### PR TITLE
added small note to RENDER_SCRIPT_CAMS

### PR DESCRIPTION
**Pull request by tabarra**
Example of the issue: using easing out of freecam when you have kashacters or.

### CODE REVIEW

**Comment by technetium-cfx** [comment]
Would be best to figure out more detail on these parameters, this doesn't really help us understand what they are for.

**Comment by tabarra** [comment]
@Poggicek /\ maybe?

File: CAM/RenderScriptCams.md [hunk 1]
```diff
@@ -27,3 +27,4 @@ NativeDB Added Parameter 6: Any p5
 * **p3**: 
 * **p4**: 
 
+Note: if you are having issues with weird camera ease start positions, try setting p3 and p4 to `true`.
```
Comment (stannum-cfx) [inline CAM/RenderScriptCams.md:30]: This line needs to be moved up to above the Parameters section to work correctly in the description. Also please remove the code blocks from the description.

**Review by stannum-cfx — CHANGES_REQUESTED** [review]

**Comment by stannum-cfx** [comment]
Closing this due to inactivity, feel free to resolve the comments above and republish this PR.

**Pull request closed without merging**